packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -203,6 +203,70 @@ enum AnimationBehavior {
 /// controllers are created in [State.initState] and disposed in
 /// [State.dispose], as described in the previous section.)
 ///
+/// {@tool dartpad --template=stateful_widget_material_ticker}
```

Comment:
As noted before, there's already a lot of example code on this class. It's not clear to me what this one is supposed to show that's not already clear from the other examples.

Reply:
Ok I will be closing this PR.